src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -93,6 +93,14 @@ export default class LevelBlock {
       this.isDestroyable = false;
       this.isTransparent = true;
     }
+    
+    if (blockType === "redstone_dust") {
+      this.isEntity = true;
+      this.isWalkable = true;
+      this.isUsable = true;
+      this.isDestroyable = true;
```

Comment:
The student will be able to place and destroy redstone dust (redstone -wire- I've been told it should be called). So I figured that was the right flag?